src/datadog/parse_util.cpp

```diff
@@ -183,15 +182,14 @@ Expected<std::unordered_map<std::string, std::string>> parse_tags(
       break;
     } else if (input[i] == ' ') {
       separator = ' ';
-      break;
     }
   }
 
   if (separator == 0) {
     goto capture_all;
   }
 
-  for (; i < end; ++i) {
+  for (i = beg; i < end; ++i) {
```

Comment:
Can we put i = 0 instead ? I'm almost sure that beg is 0 at this time and it makes the code a slightly more readable

Reply:
Make sens. Unfortunately, this PR has been opened for too long, and revisiting it for a nit is not a priority. However, please don’t hesitate to open a new PR if you'd like to revisit or continue the work